db/migrate/20190904161424_add_columns_to_tweets.rb

```diff
@@ -0,0 +1,9 @@
+class AddColumnsToTweets < ActiveRecord::Migration
+  def change
+    change_table :tweets do |t|
+      t.string :twitter_account_id
+      t.string :stolen_record_id
```

Comment:
Why strings rather than references? I think they were strings originally because I suggested making any external ids strings, but since this is internal, I think it might make more sense to make them standard references with index.

Reply:
Good call thanks — Updated in 55a014c8